cmd/kubeadm/app/util/runtime/runtime_windows.go

```diff
@@ -0,0 +1,38 @@
+// +build windows
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	winio "github.com/Microsoft/go-winio"
+)
+
+const (
+	dockerSocket     = "\\\\.\\pipe\\docker_engine"         // The Docker socket is not CRI compatible
+	containerdSocket = "\\\\.\\pipe\\containerd-containerd" // Proposed containerd named pipe for Windows
```

Comment:
My bad on not being clear enough in my previous comment here.
My concern was that the `DialPipe` below may not be able to use the path if an unix path separator is used.
Changing it to Windows one is not good, since (if detected) this one may be paired with `DefaultUrlScheme` to produce an URI and Windows path separators are not valid in there.

Reply:
Reverted back to forward slashes. DialPipe can use it